grafana-cloud-integration-utils/util.libsonnet

```diff
@@ -207,6 +208,71 @@ local integration_version_panel(version, statusPanelDataSource, height, width, x
         ],
     },
 
+  // Adds asserts specific variables to the dashboards
+  add_asserts_variables(dashboard, config)::
+    if std.member(config.statusPanelsDashboardsMetrics, dashboard.title) then
+      {
+        templating: {
+          list: [
+            var.query.new('env')
+            + var.query.withDatasource('prometheus', config.statusPanelsDatasourceNameMetrics)
+            + var.query.queryTypes.withLabelValues(
+              'asserts_env',
+              'asserts:mixin_workload_job',
+            )
+            + var.query.generalOptions.withLabel('Asserts environment')
+            + var.query.selectionOptions.withIncludeAll(
+              value=true,
+              customAllValue='.+'
```

Comment:
One question, how would it handle situations when there is no asserts_env label at all? (for cases where integration ared used without asserts?) 
Would `.*` be better?

Reply:
Yes as suggested by you on the call today, `.*` would be better. Updating.

Reply:
LGTM